### ISSUE TITLE
Mechanic’s hands

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10598]
**Preferred term label:**

**Synonyms**
hyperkeratosis of the hands

hyperkeratotic hands

**Definition (free text, please give PubMed ID)**
https://understandingmyositis.org/diagnose-myositis/mechanics-hands/ : **Mechanic’s hands** is a roughening and cracking of the skin of the tips and sides of the fingers, resulting in irregular, dirty-appearing lines that resemble those of a mechanic or manual laborer.

PMID: 32284267: Mechanic's hands are fissured, erythematous, hyperkeratotic eruptions on the hands, resembling the type of occupational dermatosis that might be seen in a manual laborer. The lesions are classically on the lateral edges of the fingers and thumbs, largely sparing the palmar, and dorsal aspects of the digits. Recent studies have found that mechanic's hands are seen in 16-21% of ASyS patients.

PMID: 36186825 Mechanics hands refer to roughening and cracking of the skin of the lateral aspects of the fingers, typically the radial
border of the index and ulnar border of the thumb. This was reported in 38% of ASyS cases included in the EuroMyositis registry (20). Mechanics hands do not appear to have any prognostic value and are also seen in DM and OM, albeit less frequently. On biopsy, an interface psoriasiform dermatitis is seen.

**Parent term (use hpo.jax.org/app)**
Palmar hyperkeratosis HP:0010765

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

ORPHA:81 Antisynthetase syndrome
**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001013
- label: Mechanic's hands
- parents: [Palmar hyperkeratosis (HP:0010765)](https://hpo.jax.org/app/browse/term/HP:0010765)
- pmid: PMID:32284267|PMID:36186825
- def: Mechanic's hands are fissured, erythematous, hyperkeratotic eruptions on the hands, resembling the type of occupational dermatosis said to resemble the hands of a manual laborer. The lesions are classically on the lateral edges of the fingers and thumbs, largely sparing the palmar, and dorsal aspects of the digits.
- comment: This feature can be observed in antisynthetase syndrome (ASyS), an autoimmune disease clinically manifested most often by interstitial lung disease, myositis, and arthritis.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10598
- synonyms:
